asab/metrics/native.py

```diff
@@ -40,3 +46,14 @@ def _on_flushing_event(self, event_name=None):
 		except FileNotFoundError:
 			pass
 			# L.warning("File '/proc/self/status' was not found, skipping reading native metrics.")
+
+
+class MetricsLoggingHandler(logging.Handler):
+
+	def emit(self, record):
+		level = record.levelno
+
+		if level == 30:
```

Comment:
`if level >= 30 and level < 40:` ...

Reply:
According to the formatter it is the (25, 30> interval: https://github.com/TeskaLabs/asab/blob/master/asab/log.py#L234

Reply:
good spot, keep it aligned then, please.